src/components/Home.vue

```diff
@@ -405,7 +405,7 @@ export default {
                     this.selectedAddress = addr
                 }, 0)
             }
-            this.updateSelectedAddress(addr)
+            this.updateSelectedAddress({address: addr, balance: this.balance[addr]})
             this.getBalance(addr)
```

Comment:
Also delete Line 409 . The balance updating is already triggered in `this.updateSelectedAddress`.